src/raylib/textures/Image.cpp

```diff
@@ -20,13 +20,31 @@ namespace raylib
 
         Image::~Image() { unload(); }
 
-        void Image::load() { _image = LoadImage(_fileName.generic_string().c_str()); }
+        void Image::load()
+        {
+            _image = LoadImage(_fileName.generic_string().c_str());
+            _loaded = true;
+        }
 
-        void Image::loadFromTexture(const Texture2D &texture) { _image = LoadImageFromTexture(texture); }
+        void Image::loadFromTexture(const Texture2D &texture)
+        {
+            _image = LoadImageFromTexture(texture);
+            _loaded = true;
+        }
 
-        void Image::loadFromScreen(void) { _image = LoadImageFromScreen(); }
+        void Image::loadFromScreen(void)
+        {
+            _image = LoadImageFromScreen();
```

Comment:
Maybe add a check verifying if an image is already loaded to avoid loading two images and breaking the raylib